public/app/features/profile/ChangePasswordPage.tsx

```diff
@@ -36,7 +36,9 @@ export function ChangePasswordPage({ loadUser, isUpdating, user, changePassword
       <Page.Contents isLoading={!Boolean(user)}>
         {user ? (
           <>
-            <h3 className="page-heading">Change Your Password</h3>
+            <Page.OldNavOnly>
+              <h3 className="page-sub-heading">Change Your Password</h3>
+            </Page.OldNavOnly>
```

Comment:
this fixes issue with double titles, as this page is part of the nav model and the Page component will handle the page title